docs/platforms/javascript/common/data-management/data-collected/index.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. This page lists data categories that the Sentry Android SDK collects.
+
+Many of the categories listed here require you to enable the <PlatformLink to="/configuration/options/#sendDefaultPii">sendDefaultPii option</PlatformLink>.
+
+## HTTP Headers
+
+By default, the Sentry SDK doesn't send any HTTP response or request headers.
```

Comment:
this isn't entirely true, right? IIRC from your PR, it's true for browser but in Node we always send headers? If so, let's maybe only show this for browser JS?

Reply:
yeah this is currently not true but I was already writing the optimistic version of the docs for after [my PR](https://github.com/getsentry/sentry-javascript/pull/16143) is merged :D

But since we don't filter sending the HTTP headers I'll rewrite this